assets/reviews.js

```diff
@@ -107,7 +107,7 @@ const setupReviews = async () => {
   }
 
   try {
-    const res = youcanjs.product.fetchReviews(productId, { limit: 3 });
+    const res = youcanjs.product.fetchReviews(productsId, { limit: 3 });
```

Comment:
why productsId? it should stay productId i believe

Reply:
we need a different name for the product reviews section so we don't have any confusion between the one as a block, to avoid bugs